web/modules/mof/src/ModelEvaluator.php

```diff
@@ -211,12 +206,42 @@ public function getTotalProgress(): float {
   }
 
   /**
-   * Class 3 has a conditional pass if these components have an open source license.
+   * Class 3 has a conditional pass if these components have an open source license but we will
+   * inform the user that a type-appropriate license should be used.
    *  - `Model parameters (Final)` (10)
+   *  - `Technical report` (11)
+   *  - `Evaluation results` (12)
+   *  - `Model card` (13)
+   *  - `Data card` (14)
    */
+  public function getConditionalMessage(): string {
+    $licenses = $this->model->getLicenses();
+    $msg = 'This model has an open source license on the following components, it should be using a type-appropriate license:';
+    if (isset($licenses[10]) && $this->licenseHandler->isOpenSource($licenses[10]['license'])) {
+      $msg = $msg . '<br>' . '- Model parameters (Final) of type data.';
+    }
+    if (isset($licenses[11]) && $this->licenseHandler->isOpenSource($licenses[11]['license'])) {
+      $msg = $msg . '<br>' . '- Technical report of type documentation.';
+    }
+    if (isset($licenses[12]) && $this->licenseHandler->isOpenSource($licenses[12]['license'])) {
+      $msg = $msg . '<br>' . '- Evaluation results of type documentation.';
+    }
+    if (isset($licenses[13]) && $this->licenseHandler->isOpenSource($licenses[13]['license'])) {
+      $msg = $msg . '<br>' . '- Model card of type documentation.';
+    }
+    if (isset($licenses[14]) && $this->licenseHandler->isOpenSource($licenses[14]['license'])) {
+      $msg = $msg . '<br>' . '- Data card of type documentation.';
+    }
+    return $msg;
```

Comment:
@gregcube If you know of a better way to achieve the same result, please, let me know. I'm surely showing my lack of expertise in Drupal (I have none!) by reverting to using HTML markup at this level...

Reply:
No problem. I can fix this up.